templates/DataDiscovery/index.html.twig

```diff
@@ -85,6 +85,9 @@
                             <td valign="top">
                                 <input type="button" id="clear-button" class="button" value="Clear/Show All" onclick="clearAll();">
                             </td>
+                            <td valign="top">
+                                <input type="button" id="searchlink-button" class="button" value="Search" onclick="parent.location=Routing.generate('pelagos_app_ui_searchpage_default')">
```

Comment:
It's say the onClick is a nono. If you are going to use jQuery, generate the link in javascript.

Reply:
And also would use window, rather than parent.